src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_MAX_LENGTH = 10;
+    private K[] keysStore;
+    private V[] valuesStore;
+    private int length;
+
+    public StorageImpl() {
+        this.keysStore = (K[]) new Object[ARRAY_MAX_LENGTH];
+        this.valuesStore = (V[]) new Object[ARRAY_MAX_LENGTH];
+        length = 0;
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < length; i++) {
+            if ((keysStore[i] == null && key == null)
```

Comment:
```suggestion
            if (keysStore[i] ==  key 
```
good way to compare nulls